spec/sms_sender_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe SMSSender do
+  
```

Comment:
I won't leave any comments here because it looks like you paused work on this (and I'm likely to end up stating the obvious).